src/helpers/Activity_Wheel.java

```diff
@@ -1,60 +1,115 @@
+package helpers;
+
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.Random;
-import javax.swing.JOptionPane;
+import java.util.Scanner;
+
 public class Activity_Wheel {
-    public static void pickActivity() {
-        //variable "dice" acts as the randomizer depending on the int (number) given (shown later on)
+
+    private static String activityMessage; // may not need this anymore
+
+    /**
+     * Returns a String message containing an activity that is randomly selected from an ArrayList
+     * of possible activities, which is filled based on the weather String passed to the method
+     *
+     * @param weather a String containing a type of weather condition obtained from the weatherAPI
+     * @return a String message
+     * @throws FileNotFoundException
+     */
+    public static String generateActivity(String weather) throws FileNotFoundException {
+        
         Random dice = new Random();
+        ArrayList<String> activities = null;
+
+        switch (weather) {
+            case "Clear":
+                activities = generateList(new File("ref/Clear.txt"));
+                break;
+            case "Clouds":
+                activities = generateList(new File("ref/Clouds.txt"));
+                break;
+            case "Rain", "Drizzle":
+                activities = generateList(new File("ref/Rain.txt"));
+                break;
+            case "Thunderstorm":
+                activities = generateList(new File("ref/Storm.txt"));
+                break;
+            case "Snow":
+                activities = generateList(new File("ref/Snow.txt"));
+                break;
+            default:
+                System.out.println("error");
+                break;
+        }
+        return "You should " + activities.get(dice.nextInt(activities.size()));
+    }
+
+    /**
+     * Fills an ArrayList of Strings with the lines of a txt file corresponding to a type of weather
+     *
+     * @param file the reference file
+     * @return an ArrayList containing different activities
+     * @throws FileNotFoundException
+     */
+    private static ArrayList<String> generateList(File file) throws FileNotFoundException {
+        Scanner fileReader = new Scanner(file);
+        ArrayList<String> activities = new ArrayList<>();
+
+        while (fileReader.hasNext()) {
+            activities.add(fileReader.nextLine());
+        }
+        return activities;
+    }
 
-        //this will be changed to get the information of the weather condition the beacon detects
-        //raining is just used to show that the code runs and uses the "sunny method"
-        //rest of notes are shown in the sunny method as side methods run similarly
-        String weather = "sunny";
+    // Consolidated everything below into the 2 methods above
+
```

Comment:
We might not need anything below this if we decide on keeping the txt file idea, if we want the option to add activities as Sal suggested we could just use FileWriter